.github/workflows/lint.yml

```
@@ -0,0 +1,23 @@
+name: Lint
+
+on:
```

yamllint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)